### PR TITLE
Show CPU core temp on Setup tab

### PR DESCRIPTION
**Pull request by HThuren**
From API 1.46, get CPU temperature from betaflight firmware by new record in MSP_STATUS_EX
Depends of betaflight PR 12608

Also read configState flags, found in betaflight msp.c, but not read in betaflight_configurator MSPHelper.js
This record are older than Nov 16, 2019

Screenshot, temperature at last line:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/msp/MSPHelper.js
Comment (haslinghuis) [inline src/js/msp/MSPHelper.js:217]: Nice to have !

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
> If OSD units is imperial, then Fahrenheit maybe?

well well, but are a setting for OSD, cli status command give C, soee
I suggest this as a next step, a setting i configurator controlling unit :-)

**Comment by nerdCopter** [comment]
> > If OSD units is imperial, then Fahrenheit maybe?
> 
> well well, but are a setting for OSD, cli status command give C, soee I suggest this as a next step, a setting i configurator controlling unit :-)

that's fair, no need to add another betaflight setting

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
